be/src/olap/rowset/segment_v2/column_reader.h

```diff
@@ -305,9 +307,12 @@ class DefaultValueColumnIterator : public ColumnIterator {
     std::string _default_value;
     bool _is_nullable;
     FieldType _type;
+    size_t _length;
```

Comment:
pls add some comments for length and value_size. This may cause people confused

Reply:
Done. Rename these two variables.